src/robusta/cli/main.py

```diff
@@ -29,6 +29,10 @@
     replace_in_file,
 )
 
+FORWARDER_CONFIG_FOR_SMALL_CLUSTERS = "256Mi"
+RUNNER_CONFIG_FOR_SMALL_CLUSTERS = "512Mi"
+GRAFANA_CONFIG_FOR_SMALL_CLUSTERS = "256Mi"
```

Comment:
I think 64MB should be enough here as well. (I just looked at the memory chart for both on small clusters, and it was less than 64MB)
In addition, I would name it GRAFANA_RENDERER , cause grafana is a different pod. might be confusing

Reply:
when you mean for both do you mean robusta runner too?

Reply:
No, The runner does need 512MB
The renderer and forwarder will be able to work with 64MB